src/successModal/SuccessModal.style.tsx

```diff
@@ -81,6 +81,9 @@ const Content = styled.div`
 
   @media (${responsiveBreakpoints.isMediaLarge}) {
     height: auto;
+    /* gutter */
+    margin-left: ${space.l};
+    max-width: 537px; /*Magic number from spec*/
```

Comment:
Hmm it's hard to tell if this is the value we should rely on because they use an arbitrary screen size.
Can you double check with design?

Reply:
Checked with design and used component sizing instead